lib/modules/apostrophe-browser-utils/public/js/lean.js

```diff
@@ -87,42 +87,30 @@
   // Implementation detail of `apos.utils.post` and `apos.utils.get`
   function monitor(xmlhttp, callback) {
     xmlhttp.addEventListener("load", function() {
-      // Grab the response header;
       var responseHeader = this.getResponseHeader("Content-Type");
-      // Trim the response so we can accurately test;
       var trimResponse = this.responseText.trim();
       var data;
-      // If the response header is `application/json`, immediately try
-      // to JSON.parse(), and if it fails, report the error.
       if (responseHeader === 'application/json; charset=utf-8') {
         try {
           data = JSON.parse(this.responseText);
         } catch (e) {
           return callback(e);
         }
-      // If there is no explicit content-type header,
-      // test the trimmed response against our isJson() method.
-      // If it returns `true`, try to JSON.parse() it.
-      // And if it fails, try to JSON.stringify() it, and if that fails,
-      // report an error.
       } else {
-        if (isJSON(trimResponse, callback)) {
+        if (maybeJson(trimResponse, callback)) {
           try {
             data = JSON.parse(this.responseText);
           } catch (e) {
             try {
-              data = JSON.stringify(this.responseText);
+              data = this.responseText;
             } catch (e) {
               return callback(e);
```

Comment:
Just `data = this.responseText` is very unlikely to throw an exception, I would not try/catch it. I think `responseText` is guaranteed to exist.